docs/users-grants.rst

```diff
@@ -0,0 +1,169 @@
+    >>> builder.set_role('psc:role::BETA_SYSTEM_ADMIN')
+    <cbc_sdk.platform.users.User.UserBuilder object at 0x00000209B8123D00>
+    >>> builder.build()
+
+Alternately, you may construct a *template object* (a Python ``dict``) that contains the user's information and
+create the user directly.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> user_template = {'first_name': 'Selyse', 'last_name': 'Florent', 'email': 'sflorent@example.com',
+    ...                  'phone': '877-555-9099', 'role_urn': 'psc:role::BETA_SYSTEM_ADMIN'}
+    >>> User.create(api, user_template)
+
+**Note:** A user that has just been created will *not* be visible in either the UI or in a ``User`` query as detailed
+above, until the intended user has responded to their invitation E-mail message, activated the account, and set
+a password.
+
+User Access Grants
+------------------
+
+Every user object has an *access grant* object associated with it, defining the access roles they are permitted to use.
+You can use the ``grant()`` method on a ``User`` to get the grant and inspect or modify it.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> user = api.select(User, 2345672)
+    >>> print(f"{user.first_name} {user.last_name}")
+    Arianne Martell
+    >>> grant = user.grant()
+    >>> print(grant.roles)
+    ['psc:role::BETA_SUPER_ADMIN']
+    >>> grant.roles = ['psc:role::BETA_SYSTEM_ADMIN']
+    >>> grant.save()
+    <cbc_sdk.platform.grants.Grant: id psc:user:1A2B3C4DE:2345672> @ https://defense.conferdeploy.net
+    >>> print(grant.roles)
+    ['psc:role::BETA_SYSTEM_ADMIN']
+
+You can see what roles you can work with by using the ``get_permitted_role_urns()`` function:
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import Grant
+    >>> for index, role_urn in enumerate(Grant.get_permitted_role_urns(api)):
+    ...     print(f"{index}. {role_urn}")
+    ...
+    0. psc:role::BETA_LEVEL_3_ANALYST
+    1. psc:role::KUBERNETES_SECURITY_DATAPLANE_ONLY
+    2. psc:role::ALL_AND_LR
+    3. psc:role::BETA_LEVEL_1_ANALYST
+    4. psc:role::BETA_SYSTEM_ADMIN
+    5. psc:role::KUBERNETES_SECURITY_DATAPLANE
+    6. psc:role::VIEW_ONLY
+    7. psc:role::ALL
+    8. psc:role::KUBERNETES_SECURITY_ADMIN_USER
+    9. psc:role::BETA_SUPER_ADMIN
+    10. psc:role::KUBERNETES_SECURITY_READ_ONLY_USER
+    11. psc:role::CONTAINER_IMAGE_CLI_TOOL
+    12. psc:role::KUBERNETES_SECURITY_DEVOPS
+    13. psc:role::BETA_VIEW_ALL
+    14. psc:role::KUBERNETES_SECURITY_DEVOPS_VIEW_ONLY
+    15. psc:role::BETA_LEVEL_2_ANALYST
+    16. psc:role::KUBERNETES_SECURITY_DEVELOPER
+
+Some organizations employ *access profiles* on the access grants, which allow roles for a user to be specified for
```

Comment:
This is the recommendation approach over assigning a role directly to a grant. I believe by default the UI will use profiles. Would it be possible to add an example explaining the benefit and features to a profile over assigning a role to a grant directly

Reply:
Can you help me figure out what that should look like? Because I'm not entirely clear on what the actual benefits are here, and don't know what an example dealing with profiles should look like.

Reply:
I added two examples of adding a new profile to a grant: one with the builder, one with a template.